### ISSUE TITLE
unexpected EOF [ERROR] root: eval: *terraform.EvalSequence, err: 1 error(s) occurred

### ISSUE THREAD

**Issue by hashibot** [terraform-providers/terraform-provider-vsphere#73]
_This issue was originally opened by @marrotte as hashicorp/terraform#11525. It was migrated here as part of the [provider split](https://www.hashicorp.com/blog/upcoming-provider-changes-in-terraform-0-10/). The original body of the issue is below._

<hr>

### Terraform Version
Terraform v0.8.5

### Affected Resource(s)
- vsphere_virtual_machine

### Terraform Configuration Files
```variable "vsphere_password" {}

# Configure the VMware vSphere Provider
provider "vsphere" {
  user           = "administrator@vsphere.local"
  password       = "${var.vsphere_password}"
  vsphere_server = "192.168.38.31"
  # if you have a self-signed cert
  allow_unverified_ssl = true
}


# Create a virtual machine 
resource "vsphere_virtual_machine" "ubuntu-docker" {
  count  = 3
  name   = "ubuntu-docker-master-${count.index}"
  vcpu   = 2
  memory = 4096

  network_interface {
    label = "VM Network"
  }

  network_interface {
    label = "vchPortGroup"
  }

  disk {
    template = "Templates/ubuntu-1604-docker"
    datastore = "datastore1"
  }
}
```

### Debug Output
https://gist.github.com/marrotte/b687a700bce33d1987a8bdd64d379313

### Panic Output
https://gist.github.com/marrotte/9c7b77d27cb8a14bd78a14981413407c

### Expected Behavior
Built 3 vm's on esxi host and report success

### Actual Behavior
Built 3 vm's on esxi host and report failure

### Steps to Reproduce
`terraform apply`

**Comment by carlossg** [comment]
this will likely be fixed with #96

**Comment by vancluever** [comment]
Hello! 

This issue details items mainly relevant to the vSphere provider before the 1.x release. Since the release of the vSphere 1.x release, several features have been added and several have been fixed, namely having to do with the features and functionality of the `vsphere_virtual_machine` resource.

As there is a very good chance that any problems or requests described in this issue have been corrected or implemented, we are closing this issue now. However, if you are still having problems after updating your Terraform and vSphere providers to the latest version, please open a new issue.

For more details on the state of the provider now, please check out these links:

[vSphere Provider Documentation Index](https://www.terraform.io/docs/providers/vsphere/index.html)
[`vsphere_virtual_machine` Resource Documentation](https://www.terraform.io/docs/providers/vsphere/r/virtual_machine.html)
[Blog post detailing an end-to-end setup using the 1.x releases](https://www.hashicorp.com/blog/a-re-introduction-to-the-terraform-vsphere-provider)

Thanks!
